deid/data/__init__.py

```diff
@@ -11,32 +11,4 @@
 
 import os
 
-from deid.logger import bot
-from deid.utils import get_installdir
-
 data_base = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Is this data_base used anywhere? E.g., why are we keeping it?

Reply:
I think it's still used to get `deid.dicom` for example, right ?

Reply:
Right - I was thinking those deid.dicom would stay with the library, and just the larger datasets would be externally installed.

Reply:
Yes, it totally makes sense to keep this file!